Dte.Common/Content/CustomMessageEmail.cs

```diff
@@ -1,429 +1,48 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
 namespace Dte.Common.Content
 {
     public static class CustomMessageEmail
     {
-        // TODO - html file or S3, make the text replaceable so can replace for difference trigger sources.
-        public static string GetCustomMessageHtml()
+        private const string BodyPlaceholder = "###BODY_REPLACE###";
+        private const string EmailTemplateResourceSuffix = "EmailTemplate.html";
+        private static readonly Lazy<string> CachedTemplate = new Lazy<string>(LoadTemplate);
+
+        public static string GetCustomMessageHtml(string bodyContent)
         {
-            return @"<!DOCTYPE html>
-<html
-  xmlns=""http://www.w3.org/1999/xhtml""
-  xmlns:v=""urn:schemas-microsoft-com:vml""
-  xmlns:o=""urn:schemas-microsoft-com:office:office""
->
-<head>
-    <title></title>
-    <!--[if !mso]>
-    <meta http-equiv=""X-UA-Compatible"" content=""IE=edge"" />
-    <![endif]-->
-    <meta http-equiv=""Content-Type"" content=""text/html; charset=UTF-8"" />
-    <meta name=""viewport"" content=""width=device-width, initial-scale=1.0"" />
+            // Use cached template if available
+            string template = CachedTemplate.Value;
 
-    <!--[if mso]>
-    <xml>
-        <o:OfficeDocumentSettings>
-            <o:AllowPNG /> <o:PixelsPerInch>96</o:PixelsPerInch>
-        </o:OfficeDocumentSettings>
-    </xml>
-    <![endif]-->
+            if (string.IsNullOrEmpty(template))
```

Comment:
Exception already thrown if you can't load the template. This check is unnecessary.